paraview/xmls/PersistenceDiagramDictionary.xml

```diff
@@ -0,0 +1,303 @@
+        number_of_elements="1"
+        default_values="0"
+        panel_visibility="advanced">
+        <BooleanDomain name="bool"/>
+        <Documentation>
+            Activate progressive approach
+        </Documentation>
+      </IntVectorProperty>
+
+
+        <IntVectorProperty
+        name="StopCondition_"
+        command="SetStopCondition_"
+        label="Stopping condition"
+        number_of_elements="1"
+        default_values="1"
+        panel_visibility="advanced">
+        <BooleanDomain name="bool"/>
+        <Documentation>
+            Activate the interrupt condition
+        </Documentation>
+      </IntVectorProperty>
+
+        <IntVectorProperty
+        name="sortedForTest_"
+        command="SetsortedForTest_"
+        label="sort for testing"
+        number_of_elements="1"
+        default_values="0"
+        panel_visibility="advanced">
+        <BooleanDomain name="bool"/>
+        <Documentation>
+            Sort each diagrams by persistence for test
+        </Documentation>
+      </IntVectorProperty>
+
+        <IntVectorProperty
+        name="CreationFeatures_"
+        command="SetCreationFeatures_"
+        label="Creation of features"
+        number_of_elements="1"
+        default_values="1"
+        panel_visibility="advanced">
+        <BooleanDomain name="bool"/>
+        <Documentation>
+            Performs the creation of features
+        </Documentation>
+      </IntVectorProperty>
+
+      <IntVectorProperty
+        name="CompressionMode_"
+        command="SetCompressionMode_"
+        label="Compression Mode"
+        number_of_elements="1"
+        default_values="1"
+        panel_visibility="advanced">
+        <BooleanDomain name="bool"/>
+        <Documentation>
+            Activate compression mode
+        </Documentation>
+      </IntVectorProperty>
+
+      <IntVectorProperty
+        name="DimReductMode_"
+        command="SetDimReductMode_"
+        label="Dimension Reduction Mode"
+        number_of_elements="1"
+        default_values="0"
+        panel_visibility="advanced">
+        <BooleanDomain name="bool"/>
+        <Documentation>
+            Activate dimension reduction mode
+        </Documentation>
+      </IntVectorProperty>
+
+
+      <IntVectorProperty name = "MaxEpoch_" label = "Max Epoch" command="SetMaxEpoch_" number_of_elements = "1" default_values="1000" >
+      <IntRangeDomain name = "Range" min = "0" max = "1000" />
+      </IntVectorProperty>
+
+      <IntVectorProperty name = "Seed_" label = "Seed for random init" command="SetSeed_" number_of_elements = "1" default_values="0" >
+      <IntRangeDomain name = "Range" min = "0" max = "1000" />
+      </IntVectorProperty>
+
+
+      <!-- A string parameter that controls the name of the output array -->
+
+
+      <!-- Create a UI group that contains all output parameter widgets (here only one) -->
+      <PropertyGroup panel_widget="Line" label="Input Options">
+        <Property name="AtomNumber_" />
```

Comment:
Thanks for listing all the other options here too.
Also:
- please organize and group them by theme
- make the labeling convention more uniform (the first word only starts with a capital)
- remove testing options
- make the labels more explicit
- extend the option documentation (`<Documentation>...</Documentation>`)